scripts/diffmazing_cropper.sh

```diff
@@ -21,29 +21,13 @@ then
 fi
 
 
-if [ $# -eq 0 ]
-then
-  echo ""
-  echo "Usage"
-  echo "  $0 <diff files to convert>"
-  echo ""
-  echo "Example"
-  echo "  $0 ~/downloads/*.diff.png"
-  echo ""
-  exit -1
-fi
-
-
-has_convert=$(command -v convert)
-if ! [ "$has_convert" ]
-then
+has_convert=$(command -v convert) || {
```

Comment:
What is this doing? I think this will never print anything out?

Reply:
Supposedly this is the [bash way for try/catch](https://stackoverflow.com/questions/22009364/is-there-a-try-catch-command-in-bash) and it worked for me. 

Right now `has_convert=$(command -v convert)` fails silently without any messages.